pages/using_angular.md

```diff
@@ -163,11 +163,8 @@ The shorthand methods `success`, `info`, `warning` and `error` will have a defau
 
 ## Using Angular CLI
 
-<div class="alert alert-info"><i>Info: </i>
-
-Angular CLI and JHipster can be used in parallel for development, and both have their own configuration files. By default, JHipster is using its own configuration when deploying applications or when using the CI-CD sub-generator.
-
-</div>
+Angular CLI is used to build and test JHipster applications. 
+However, we added a custom Webpack configuration file in order to improve Developer Experience and User Experience.
```

Comment:
```suggestion
However, we added a custom webpack configuration file in order to improve the developer experience.
```

How? What does JHipster's webpack config do that Angular CLI doesn't?

Reply:
The main thing is that we add BrowserSync, ESLint (Angular CLI is still on TSLint for now), merging JSON translation files and also the notify plugin

Reply:
What about this wording?

> However, we added a custom webpack configuration file in order to improve the developer experience by adding BrowserSync, ESLint (Angular CLI is still on TSLint for now), merging JSON translation files. So, rather than running `ng build` you should run `npm run build`, while `ng serve` should be replaced by `npm start` .

Reply:
Well ng build and ng serve work out of the box because I pointed our custom Webpack configuration in the `angular.json` so I think the last part isn't necessary, WDYT ?

Reply:
Here is the wording i purpose

> However, we added a custom webpack configuration file in order to improve the developer experience by adding BrowserSync, ESLint (Angular CLI is still on TSLint for now), merging JSON translation files and add notifications when build has completed or failed.